idrstream_cp_env.yml

```diff
@@ -0,0 +1,30 @@
+name: idrstream_cp
+
+channels:
+- conda-forge
+
+dependencies:
+- python=3.8.13
+- jupyter=1.0.0
+- pandas=1.4.2
+- pytorch=1.11.0
+- scikit-image=0.19.3
+- pyimagej=1.2.1
+- pip=22.1.2
+- numpy
+- matplotlib
+- pandas
```

Comment:
Would it be important to match version pins between this file and `idrstream_dp_env.yml`? For example, should pandas be pinned to `1.4.2` (and others where appropriate)? This may eventually effect development for code blocks which are used by both environments.

Reply:
@d33bs I think this will be good to do a deeper look into. I see I missed some of those versions. I think it would be good to test if the programs still work even with the versions specified, it is just the environment for CP takes a VERY long time to create from my experience.

This is good to keep in mind those since I haven't had experience with environments failing and causing blocks... yet.

Reply:
I definitely agree that it would be a good idea to include specific versions here. 

A quick way to add versions to all these would be to make a new (test) conda environment and install all the packages with their conda-forge install command found at https://anaconda.org/anaconda/repo. If you are able to run the example CP scripts with this test conda environment you can use the package version shown at https://anaconda.org/anaconda/repo.